src/libsync/progressdispatcher.cpp

```diff
@@ -45,7 +45,7 @@ QString Progress::asResultString(const SyncFileItem &item)
         return QCoreApplication::translate("progress", "Deleted");
     case CSYNC_INSTRUCTION_EVAL_RENAME:
     case CSYNC_INSTRUCTION_RENAME:
-        return QCoreApplication::translate("progress", "Moved to %1").arg(item._renameTarget);
+        return QCoreApplication::translate("progress", "%1 moved to %2").arg(item._file, item._renameTarget);
```

Comment:
Can you grep the code base, I'm pretty sure we have duplication of similar messages.

Reply:
grepping for "moved to" (case insensitive) only yields https://github.com/owncloud/client/blob/90708a5b02eaae871b90abde2008435d273301bf/src/gui/folder.cpp#L499

I wouldn't change either: this one is displayed in limited space, and the 2 filenames make it quite long. The one in `folder.cpp` is in present perfect tense (passive voice) instead of past tense (active voice), just like all other messages  around it.

Reply:
I think for another release we should review all our strings...